cfgmgr/tunnelmgr.cpp

```diff
@@ -191,6 +210,13 @@ void TunnelMgr::doTask(Consumer &consumer)
             ++it;
         }
     }
+
+    if (!replayDone &&
```

Comment:
Prefer to keep in one line for this. This will help controlling the number of lines in the file growing quickly.